base/linalg/bidiag.jl

```diff
@@ -22,11 +22,33 @@ matrix with [`full`](:func:`full`). `ev`'s length must be one less than the leng
 
 **Example**
 
-```julia
-dv = rand(5)
-ev = rand(4)
-Bu = Bidiagonal(dv, ev, true) #e is on the first superdiagonal
-Bl = Bidiagonal(dv, ev, false) #e is on the first subdiagonal
+```jldoctest
+julia> dv = [1; 2; 3; 4]
+4-element Array{Int64,1}:
+ 1
+ 2
+ 3
+ 4
+
+julia> ev = [7; 8; 9]
+3-element Array{Int64,1}:
+ 7
+ 8
+ 9
+
+ julia> Bu = Bidiagonal(dv, ev, true) #e is on the first superdiagonal
+ 4×4 Bidiagonal{Int64}:
+  1  7  ⋅  ⋅
+  ⋅  2  8  ⋅
+  ⋅  ⋅  3  9
+  ⋅  ⋅  ⋅  4
+
+ julia> Bl = Bidiagonal(dv, ev, false) #e is on the first subdiagonal
+ 4×4 Bidiagonal{Int64}:
+  1  ⋅  ⋅  ⋅
+  7  2  ⋅  ⋅
+  ⋅  8  3  ⋅
+  ⋅  ⋅  9  4
```

Comment:
Perhaps replace `#e is` with `# ev is` for consistency with the rest of the docstring?